backend/src/main/kotlin/org/loculus/backend/utils/MetadataEntry.kt

```diff
@@ -64,8 +82,23 @@ fun metadataEntryStreamAsSequence(metadataInputStream: InputStream): Sequence<Me
                     )
                 }
 
+                var fastaIds: List<FastaId>? = null
+
+                if (addFastaIds) {
+                    val fastaId = record[fastaIdHeader]
+                    if (fastaId.isNullOrEmpty()) {
+                        throw UnprocessableEntityException(
+                            "A row in metadata file contains no $fastaIdHeader: $record",
+                        )
+                    }
+
+                    fastaIds = fastaId.split(Regex("[ ,]+"))
```

Comment:
Is it a good idea to split on spaces here?

Reply:
yes space is not allowed in fasta header IDs